tests/sentry/runner/commands/presenters/test_slackpresenter.py

```diff
@@ -74,8 +75,21 @@ def test_is_slack_enabled():
         ],
     }
 
-    assert responses.calls[1].response.status_code == 200
-    assert expected_json_data == json.loads(responses.calls[1].request.body)
+    assert responses.calls[0].response.status_code == 200
+    assert expected_json_data == json.loads(responses.calls[0].request.body)
+
+
+@pytest.mark.django_db
+@responses.activate
+@override_settings(OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL="https://test/", SENTRY_REGION="test_region")
+def test_slack_presenter_empty():
+    responses.add(responses.POST, "https://test/", status=200)
```

Comment:
You should not need this if you are testing that there are no response being sent.